### PR TITLE
[doc][yba] Note on dropping tables in xCluster

### PR DESCRIPTION
**Pull request by ddhodge**
Note on dropping tables in xCluster
DOC-177
DOC-288

@netlify /preview/yugabyte-platform/create-deployments/async-replication-platform/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 1]
```diff
@@ -149,6 +149,10 @@ This page allows you to do the following:
 
   - To delete a table from the replication, click **... > Remove Table**. This removes both the table and its index tables from replication. If you decide to remove an index table from the replication group, it does not remove its main table from the replication group.
 
+    {{<note title="Dropping tables">}}
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md:152]:
```suggestion
    {{<note title="Before dropping tables">}}
```

Comment (aishwarya24) [reply docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md:152]: just feel the title can sound more like a warning so the user definitely reads it.

**Review by aishwarya24 — APPROVED** [review]
LGTM with a comment

**Review by Jethro-M — APPROVED** [review]
Looks great, thanks Dwight!

**Pull request merged**